FirebaseRemoteConfig/Sources/RCNFetch.m

```diff
@@ -370,6 +370,9 @@ - (void)fetchWithUserProperties:(NSDictionary *)userProperties
           }
         }
         if (statusCode != kRCNFetchResponseHTTPStatusCodeOK) {
+          // Update metadata about fetch failure.
+          [strongSelf->_settings updateMetadataWithFetchSuccessStatus:NO];
```

Comment:
Looks like by this line the method is already called at line 362. Do we need to call this method 2nd time?

Reply:
right. Deleted.